tests/integration/prepare/test_ncep_reanalysis_surface_pr_wtr.py

```diff
@@ -0,0 +1,663 @@
+            "description": "",
+            "extent": {
+                "center_dt": "2018-01-03T00:00:00+00:00",
+                "coord": {
+                    "ll": {
+                        "lat": -91.25,
+                        "lon": -1.25
+                    },
+                    "lr": {
+                        "lat": -91.25,
+                        "lon": 358.75
+                    },
+                    "ul": {
+                        "lat": 91.25,
+                        "lon": -1.25
+                    },
+                    "ur": {
+                        "lat": 91.25,
+                        "lon": 358.75
+                    }
+                }
+            },
+            "format": {
+                "name": "netCDF"
+            },
+            "grid_spatial": {
+                "projection": {
+                    "geo_ref_points": {
+                        "ll": {
+                            "x": -1.25,
+                            "y": -91.25
+                        },
+                        "lr": {
+                            "x": 358.75,
+                            "y": -91.25
+                        },
+                        "ul": {
+                            "x": -1.25,
+                            "y": 91.25
+                        },
+                        "ur": {
+                            "x": 358.75,
+                            "y": 91.25
+                        }
+                    },
+                    "spatial_reference": (
+                        'GEOGCS["Hu Tzu Shan 1950",DATUM["Hu_Tzu_Shan_1950",SPHEROID'
+                        '["International 1924",6378388,297,AUTHORITY["EPSG","7022"]]'
+                        ',TOWGS84[-637,-549,-203,0,0,0,0],AUTHORITY["EPSG","6236"]],'
+                        'PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree"'
+                        ',0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4236"]]'
+                    )
+                }
+            },
+            "id": "e22bea968eb55e7fa770cba574c66dda",
+            "image": {
+                "bands": {
+                    "water_vapour": {
+                        "band": 9,
+                        "layer": "pr_wtr",
+                        "path": "pr_wtr.eatm.2018.test.nc"
+                    }
+                }
+            },
+            "product_name": "ncep_reanalysis_surface_pr_wtr",
+            "product_type": "auxiliary",
+            "sources": {}
+        }
+    ]
+
+    result = run_prepare_cli(
+        ncep_reanalysis_surface_pr_wtr.main,
+        '--output', str(output_path), str(NCEP_PR_WTR_FILE)
+    )
+
+    assert expected_metadata_path.exists()
+    docs = list(yaml.safe_load_all(expected_metadata_path.open()))
+
+    for idx in range(len(expected_doc)):
+        doc_diff = _diff(expected_doc[idx], docs[idx], exclude_paths="root['creation_dt']")
+        assert doc_diff == {}, pformat(doc_diff)
```

Comment:
:+1: Nice tests.

(If it becomes unwieldy: just testing the first two docs and the total count would be fine with me → less code to keep up to date)

Reply:
Mostly I just wanted to get the tests over multiple days; not sure if this is an invalid desire.